README.md

```diff
@@ -35,13 +35,10 @@ module.exports = function(defaults) {
 
 
 ## Using plugins
-- [postcss-partial-import](https://github.com/jonathantneal/postcss-partial-import),
+- [postcss-import](https://github.com/postcss/postcss-import),
 - [postcss-nested](https://github.com/postcss/postcss-nested),
-- [postcss-custom-properties](https://github.com/postcss/postcss-custom-properties),
 - [postcss-inline-svg](https://github.com/TrySound/postcss-inline-svg),
-- [postcss-svgo](https://github.com/ben-eb/postcss-svgo),
 - [autoprefixer](https://github.com/postcss/autoprefixer),
-- [postcss-hexrgba](https://github.com/seaneking/postcss-hexrgba)
```

Comment:
I have mixed feelings on removing `postcss-hexrgba`. One on hand, it mimics a Sass feature I've used heavily (Sass lets you use `rgba()` with any valid color value the first argument, then the alpha for the 2nd). On the other hand, how often are we using a plain color and just adding alpha?

Thoughts for discussion/posterity:

* For a11y reasons, we should avoid alpha anywhere color contrast needs to be measured (but of course, that's a +Design conversation). 
* There's also a big issue with colors in variables. Unlike Sass variables, CSS custom properties don't allow color manipulation functions (and native CSS color-mod is a ways away still).
* Additionally, hexa is being supported well (`#ffffff7f` = white @ 50% alpha) and our team is loving HSL more.

I guess I just convinced myself: I don't see a strong case for keeping `hexrgba`. 😄

Reply:
I agree that we should be avoiding transparencies for accessibility. This is also why I avoid color functions, being it allows for color modification that's harder to control/audit for sufficient color contrast.

I'd love to see `hsl` and `hsla` being used as standards. It's more intuitive for color systems and better aligns with design software (8 digit HEX isn't a thing in design tools).